spec/services/expense_services_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+RSpec.describe ExpenseService, type: :service do
+
+  let(:group) { create(:group) }
+  let(:split_type) { create(:split_type) }
+  let(:user) { create(:user) }
+  let(:expense_payers_attributes) { [{ paid_by: user.id, amount: 1000, date: Date.today }] }
+  let(:expense_params) { attributes_for(:expense).merge(split_type_id: split_type.id,
+                                                        expense_payers_attributes: expense_payers_attributes) }
+
+  describe '#create_expense' do
+    context 'with valid expense params' do
+      it 'creates a new expense' do
+        service = ExpenseService.new(group, expense_params)
+        result = service.create_expense
+
+        expect(result[:expense]).to be_a(Expense)
+        expect(result[:expense]).to be_persisted
+        expect(result[:errors]).to be_nil
+      end
+    end
+
```

Comment:
add tests for the newly added validations.
1. at least one payer should be present
2. split type should be of a valid value from the master data